windows-icon-handler/refresh-icon-handler/refresh-icon-handler.rgs

```diff
@@ -0,0 +1,3 @@
+HKCR
```

Comment:
Header ?

Reply:
I tried to put a header comment at the beginning of this file but is causes issues.

TL;DR this file is bundled as a ressource into `refresh-icon-handler.dll` and it's used to update windows registry when the dll is being registered by the `regsvr32` command. Putting a comment at the beginning of this file made the dll registration fail. In fact there's no comments support in this type of file as explained [here](https://stackoverflow.com/questions/11775671/what-is-the-syntax-of-a-comment-in-a-rgs-file).